.circleci/workflows.yml

```diff
@@ -3,7 +3,7 @@ parameters:
   ManualBuild:
     description: 'When `true`, pipeline will continue to build even if its been filtered out by config.yml (Will Not Run on Master Branch)'
     type: boolean
-    default: true
+    default: false
```

Comment:
This build was failing because of an issue with the PowerShell build but that isn't needed for this. @jworkmanjc recommended disabling this build for now. There will still be a failing build on the PR but it can be ignored.